### DISCUSSION TITLE
Aggiunta campi a risposta servizio R005 ricerca/consultazione/ansc/intestatario

### DISCUSSION THREAD

**Discussion by LucaBonini64** [italia/ansc#1099] [Miglioramenti]
Nel ModelSoggettoInt non vengono inseriti tutti i campi che servono per la stesura degli atti costringendo il comune ad eseguire delle ulteriori consultazioni su ANPR per reperire tutte le informazioni necessarie.
Nella fattispecie servirebbe avere i dati dell'atto di nascita, obbligatori per effettuare l'annotazione contestuale negli atti di morte e usati anche per l'identificazione degli sposi nei matrimoni e unioni civili (questi dati sono assenti anche nel ModelSoggetto usato con le version precedenti) ed i dati dello stato civile e del coniuge, anche in questo caso obbligatori per la stesura degli atti di morte.
Grazie.

**Comment by asqueglia** [comment]
Buon pomeriggio, al momento non si prevedono ulteriori estensioni dei servizi atomici realizzati. Il servizio che consente di ottenere tutti gli atti registrati in ANSC per un dato soggetto è il seguente  {{R005}}/consultazione/ansc/soggetto/1.

Si chiude pertanto la discussion
